pkg/kube/wait.go

```diff
@@ -67,13 +67,13 @@ func (c *Client) waitForResources(timeout time.Duration, created Result) error {
 				}
 				pods = append(pods, list...)
 			case *v1.Pod:
-				pod, err := kcs.Core().Pods(value.Namespace).Get(value.Name, metav1.GetOptions{})
+				pod, err := kcs.CoreV1().Pods(value.Namespace).Get(value.Name, metav1.GetOptions{})
 				if err != nil {
 					return false, err
 				}
 				pods = append(pods, *pod)
 			case *extensions.Deployment:
```

Comment:
we'll probably need to add the appsv1.* types in the watcher here as well.

Reply:
I agree with @bacongobbler, we'll need to keep these ones and add all of the `apps/v1` types here so as to not break `--wait` functionality